README.md

```diff
@@ -96,7 +96,7 @@ The list of available actions can be seen via `medic-conf --help`.
 
 _Added in v3.2.0_
 
-In order to avoid overwriting someone elses configuration medic-conf records the last uploaded configuration snapshot in the `.snapshots` directory. The `remote.json` file should be committed to your repository along with the associated configuration change. When uploading future configuration if medic-conf detects the snapshot doesn't match the configuration on the server you will be prompted to overwrite or cancel.
+In order to avoid overwriting someone else configuration medic-conf records the last uploaded configuration snapshot in the `.snapshots` directory. The `remote.json` file should be committed to your repository along with the associated configuration change. When uploading future configuration if medic-conf detects the snapshot doesn't match the configuration on the server you will be prompted to overwrite or cancel.
```

Comment:
```suggestion
In order to avoid overwriting someone else's configuration medic-conf records the last uploaded configuration snapshot in the `.snapshots` directory. The `remote.json` file should be committed to your repository along with the associated configuration change. When uploading future configuration if medic-conf detects the snapshot doesn't match the configuration on the server you will be prompted to overwrite or cancel.
```

`In order to avoid overwriting someone else configuration` is grammatically incorrect.

Reply:
Thanks! fixed